lib/registries/addon/components/registries-states/component.ts

```diff
@@ -81,9 +89,17 @@ export default class RegistriesStates extends Component {
         return registeredFromId && pathJoin(baseURL, registeredFromId);
     }
 
-    @computed('registration.{userHasAdminPermission,state,isRoot}')
+    @computed('registration.{userHasAdminPermission,state,isRoot}', 'isModeratorMode')
```

Comment:
Am I missing something? `registration.state` cp is not a thing anymore.
```suggestion
    @computed('registration.{userHasAdminPermission,machineState,isRoot}', 'isModeratorMode')
```